Week-1/Exercise-1.2/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Exercise-1.2</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="src/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css"
```

Comment:
try using boostrap library, which are cdn initiated, for optimisation.

Reply:
How to use the line awesome icons with Bootstrap ? 
I'm not able to get it. @arqam-pesto

Reply:
Replaced this with bootstrap cdn